blockchains/ethereum/assets/0x3383c5a8969Dc413bfdDc9656Eb80A1408E4bA20/info.json

```diff
@@ -0,0 +1,20 @@
+{
+    "name": "Wrapped Anatha Token",
+    "website": "https://anatha.io",
+    "source_code": "https://github.com/anathatech/",
+    "short_description": "Reward token for the Anatha ecosystem.",
+    "description": "Reward token for the Anatha ecosystem.",
+    "socials": [
+        {
+            "name": "Twitter",
+            "url": "https://twitter.com/anatha_io",
+            "handle": "Anatha_IO"
+        },
+        {
+            "name": "Reddit",
+            "url": "https://www.reddit.com/r/Anatha",
+            "handle": "Anatha"
+        }
+    ],
+    "explorer": "https://etherscan.io/address/0x3383c5a8969dc413bfddc9656eb80a1408e4ba20"

```

Comment:
Please update the value of the `explorer` to `https://etherscan.io/token/0x3383c5a8969dc413bfddc9656eb80a1408e4ba20`